packages/dapp/src/components/CreateNewsroom.tsx

```diff
@@ -18,35 +19,45 @@ class CreateNewsroom extends React.Component<CreateNewsroomProps, CreateNewsroom
     super(props);
     this.state = {
       name: "",
+      multisig: false,
       error: "",
     };
   }
 
   public render(): JSX.Element {
-    console.log("this.props.history: " + this.props.history);
+    console.log("this.props.history:", this.props.history);
     return (
       <PageView>
         <ViewModule>
           <ViewModuleHeader>Create Newsroom</ViewModuleHeader>
           {this.state.error}
-          <input onChange={this.onChange} />
+          <input name="name" onChange={this.onChange} />
           <TransactionButton
             transactions={[{ transaction: this.createNewsroom, postTransaction: this.onNewsroomCreated }]}
           >
             Deploy Newsroom
           </TransactionButton>
+          <br />
+          <input type="checkbox" name="multisig" checked={this.state.multisig} onChange={this.onChange} /> multisig
         </ViewModule>
       </PageView>
     );
   }
 
   private onChange = (e: any) => {
-    this.setState({ name: e.target.value });
+    const target = e.target;
+    const value = target.type === "checkbox" ? target.checked : target.value;
+    const name = target.name;
+    this.setState({ [name]: value });
   };
 
   private createNewsroom = async (): Promise<TwoStepEthTransaction<any>> => {
     const civil = getCivil();
-    return civil.newsroomDeployNonMultisigTrusted(this.state.name);
+    if (this.state.multisig) {
```

Comment:
fwiw, once we have the multisig version working we probably won't provide a flow for non-multisig

Reply:
Yep I figured, just didn't want to hardcode it in yet while it's not really working.